plugins/koa/KoaIntegrationMiddleware.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+const compose = require('koa-compose');
+const Router = require('koa-router');
+const ETag = require('koa-etag');
+const assert_internal = require('reassert/internal');
+const router = new Router();
+const reconfig = require('@brillout/reconfig');
+
+module.exports = {
```

Comment:
Let's directly export `middleware()`. We don't need `multiple` and `name`.

Reply:
Done